node/task.ts

```diff
@@ -1157,6 +1159,27 @@ export function globFirst(pattern: string): string {
     return matches[0];
 }
 
+export function findFiles(includePattern: string[], excludePattern: string[], workingDir?: string): string[] {
```

Comment:
Can you point me to the tests which shows the existing pattern? 
I looked at from Task owners perspective, they expect to simply pass VSTS filters and expects the files to be discovered.